CDCgov/prime-reportstream
# 11504 - About nav

This PR ...

Adds dropdown to the subpages of About navigation

Test Steps:

1. Go to RS
2. View About in navigation and click
3. see sub items

## Changes
![Screenshot 2023-09-27 at 1 15 26 PM](https://github.com/CDCgov/prime-reportstream/assets/102491809/85524e36-555e-4736-b112-0f4614222fdd)


## Checklist

### Testing


## Linked Issues
- Fixes #11504
